docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -2,31 +2,93 @@
 Developing APBs Locally
 -----------------------
 
-Create a cluster pointing at your own docker organisation, this will
-also copy all the existing APBs in feedhenry to the dockerhub_org:
+The document is documenting best practises for the FeedHenry APB development! Some general information about the Ansible Service Broker (ASB) & Ansbile Playbook Bundles (APB) can be found link:https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/ansible_service_broker.html[here].
 
+## Installation
+
+The `apb` CLI can be installed locally, _however_ it is highly recommended using it wrapped in a Linux container. Below is an example on how to create an alias for the _wrapped_`apb` CLI:
+
+....
+alias apb='docker run --rm --privileged -v $PWD:/mnt -v $HOME/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $UID docker.io/ansibleplaybookbundle/apb'
+....
+
+
+## Creating an APB
+
+There is a link:https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/docs/getting_started.md[great tutorial] on how to write ABPs, it's highly recommended going through that!
+
+_NOTE:_ It's a good practise to have the APB end with `-apb`!
```

Comment:
I think this should be: It's required to have the APB end with `-apb`!

Reply:
Nope :-) 

I asked on their IRC channel, here is the log:
```
<matzew> question: must an "APB" end with "-apb" ? Or is that just a nice convention ? 
<rhallisey> matzew, It doesn't have to. It's just convenient for whitelisting
<rhallisey> by default, we look for -apb
```